.github/actions/checkDeployBlockers/checkDeployBlockers.js

```diff
@@ -0,0 +1,46 @@
+const core = require('@actions/core');
+const github = require('@actions/github');
+const {GITHUB_OWNER, EXPENSIFY_CASH_REPO} = require('../../libs/GithubUtils');
+
+const run = function () {
+    const octokit = github.getOctokit(core.getInput('GITHUB_TOKEN', {required: true}));
+    const issueNumber = Number(core.getInput('ISSUE_NUMBER', {required: true}));
+    let hasDeployBlockers = false;
+
+    return octokit.rest.issues.get({
+        owner: GITHUB_OWNER,
+        repo: EXPENSIFY_CASH_REPO,
+        issue_number: issueNumber,
+    })
+        .then((issue) => {
+            const body = issue.data.body || '';
+            const pattern = /-\s\[\s]/g;
+            const matches = pattern.exec(body);
+            hasDeployBlockers = matches !== null;
+
```

Comment:
If deploy blockers are found here, you should immediately log that so we know what type of deploy blocker was found (i.e: unverified PR or unresolved deploy blocker v.s: missing :shipit: ).